.github/workflows/pull-request-checks.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    types: [opened, edited, review_requested, synchronize, reopened, ready_for_review]
+
+jobs:
+  verify-description:
```

Comment:
### Example of the workflow log output:
I'm open to suggestions to improve, or feel free to submit changes against https://github.com/peternied/check-pull-request-description/tree/main if inclined

```
Found inactive checklist item New functionality includes testing.
Found inactive checklist item All tests pass
Error: Check list item that should be checked `[x]` or the item should be 'struck out' with `~` characters on both sides of the text. Line `- [ ] New functionality has been documented.
`
Found inactive checklist item New functionality has javadoc added
Found completed checklist item Commits are signed per the DCO using --signoff
Error: Check list item that should be checked `[x]` or the item should be 'struck out' with `~` characters on both sides of the text. Line `- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))
`
Error: Found 2 errors:
   - Check list item that should be checked `[x]` or the item should be 'struck out' with `~` characters on both sides of the text. Line `- [ ] New functionality has been documented.
`
   - Check list item that should be checked `[x]` or the item should be 'struck out' with `~` characters on both sides of the text. Line `- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))
`
```